src/core/typings.ts

```diff
@@ -36,3 +36,10 @@ type DeeplyMockedKeys<T> = {
   T;
 
 type MockedKeys<T> = { [P in keyof T]: jest.Mocked<T[P]> };
+
+// Need to declare like typings/index.d.ts otherwise would be overwritten
+declare module '*.svg' {
```

Comment:
Yes, I tried some ways and I suppose it's better to follow normal way in OSD. Normally we can import SVG like other plugins because of this declare. https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/typings/index.d.ts#L43 
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/core/typings.ts Because this file overwrites, so the import in core is failed.

Reply:
Would like to hear your guys thoughts, i'm okay to both ways as this is a midway solution because we will finally import from OUI.

Reply:
Why not just add to OUI or use an existing icon instead of adding code we need to clean up?

Reply:
Designer side suggests to use this icon and this icon will be added to OUI. This PR needs to catch up with other navigation changes in 2.16, in order to catch up before code freeze, we will use SVG for now and replace it once OUI supported it. I have create an issue to track and I will take this task.
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/7354